### DISCUSSION TITLE
Backpack docs

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#955] [General]
@tabacitu :)
Waiting backpack.ai take a look on these links:
https://backpackforlaravel.com/docs/6.x/crud-columns#select_from_array-1
https://backpackforlaravel.com/docs/6.x/crud-columns#select2_from_array-pro

They have exactly the same description, incomplete among other things, where is the advantage of the Pro??

**Comment by o15a3d4l11s2** [comment]
@blondie63, as the name describes, the first one is a [simple html select](https://www.w3schools.com/tags/tag_select.asp). The second one is a j[Query select2 component](https://select2.org) that comes with the additional features of select2.

**Reply by tabacitu** [reply]
Yup, this, 100%. To add to it since I see @blondie63 asked about columns (not fields)... the columns are IDENTICAL. In fact, one is including the other. The only reason they both exist is to have parity between fields and columns... each field has a column with the same name.

**Reply by o15a3d4l11s2** [reply]
Thank you for clarifying this. 90% of the time I think about fields and totally missed that the question was for the columns. 😅
